### ISSUE TITLE
Open inventory with item/slot

### ISSUE THREAD

**Issue by Nahuel1421** [MinusKube/SmartInvs#100]
I think the title already speaks for itself, what I try to do is that when the player clicks on that slot then there will be another inventory but the name of that inventory must be defined in a configuration file, what I did was that at click on the slot, close Inventory 1 and there will be inventory 2 [Which is defined in a configuration file], until there everything goes well, the problem comes when I want to return to inventory 1, since when doing the same procedure as inventory 1, instead of opening inventory 1 open inventory 2 again. If you do not understand, I will pass you my code so you can understand a little more (I will also pass the configuration file)

InventoryProvider:
https://pastebin.com/JHK3SVMa

InventoryMain(?):
https://pastebin.com/mUUUpBUY

YamlConfiguration file:
https://pastebin.com/4HrZHdPw

Thanks in advance!

**Comment by MinusKube** [comment]
Your code is quite hard to understand, but I think you'll have a big problem with this code, your "inventory" and "inventory2" variables are static, that means they are unique to the plugin, so if you have several players navigating in the inventories at the same time, I think some problems will appear.

Also, here, you're never setting the `inventory2` variable, I don't think that's intended.

**Comment by Nahuel1421** [comment]
> Your code is quite hard to understand, but I think you'll have a big problem with this code, your "inventory" and "inventory2" variables are static, that means they are unique to the plugin, so if you have several players navigating in the inventories at the same time, I think some problems will appear.
> 
> Also, here, you're never setting the `inventory2` variable, I don't think that's intended.
You are right with the static use that I will change later, the variable inventory2 is not in use so I have removed it, what I want to achieve (or that I am trying) is to be able to create the amount of inventory that the user prefers but with the smallest possible code, and it happens that the first inventory if or if it should be opened by clicking on an item that the user wants, that item is defined in a yml file that goes like this: Plugin:   Item: 'inventory name' But then if the user wants to open another inventory with some slot I would like to implement that function since it is used enough.
